### DISCUSSION TITLE
Q: Build script for macOS using macos 11

### DISCUSSION THREAD

**Discussion by GermanWarez** [letmaik/pyvirtualcam#120] [General]
The ci.yml used macos-min-version: '11.0'. Even And in the extra section for macOS 14, that uses the same version. Why is it like this, and not using '14.0'?

BTW: macOS 11 is no longer supported by Apple, so maybe it would be better to switch to 12 or 13.

**Comment by letmaik** [comment]
This is the minimum supported macOS version. In general, I try to align with what numpy is doing. You can see in their latest release the two macOS wheels:

numpy-1.26.4-cp312-cp312-macosx_11_0_arm64.whl
numpy-1.26.4-cp312-cp312-macosx_10_9_x86_64.whl
